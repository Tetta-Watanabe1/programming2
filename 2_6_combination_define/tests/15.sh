# auto=true
src=$cmdOrig.c
opt=-DVERBOSE     # コンパイルオプション
cmd=${cmdOrig}_verbose # 実行ファイル名
. compile.sh
