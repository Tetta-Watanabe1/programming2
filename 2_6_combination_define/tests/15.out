720 / 6 / 6 = 20
